eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -1,18 +1,28 @@
 #Requires -Version 7.0
 $CI_YAML_FILE = "ci.yml"
+
+#mgmt: resourceProvider to sdk package name map
+$packageNameHash = [ordered]@{"vmware" = "avs"; 
+                            "azure-kusto"="kusto";
+                            "cosmos-db"="cosmosdb";
+                            "customer-insights"="customerinsights";
+                            "monitor"="insights";
+                            "msi"="managedserviceidentity";
+                            "web"="appservice"}
```

Comment:
It's a little easier to read and update when the whitespace is consistent.
```suggestion
$packageNameHash = [ordered]@{
    "vmware" = "avs"; 
    "azure-kusto" = "kusto";
    "cosmos-db" = "cosmosdb";
    "customer-insights" = "customerinsights";
    "monitor" = "insights";
    "msi" = "managedserviceidentity";
    "web" = "appservice"
}
```

Reply:
updated